### ISSUE TITLE
Batch editing

### ISSUE THREAD

**Issue by raq929** [freedomofpress/pressfreedomtracker.us#585]
From @petersterne: I would also like to be able to edit, add tags, and publish multiple posts at the same time.

**Comment by harrislapiroff** [comment]
I agree with six years ago @conorsch—this feature is too broad to implement, unless a current editor wants it and we can chat spec
